toml/src/main/kotlin/org/rust/toml/completion/CargoTomlDependencyFeaturesCompletionProvider.kt

```diff
@@ -35,9 +37,14 @@ class CargoTomlDependencyFeaturesCompletionProvider : CompletionProvider<Complet
         val pkgName = containingArray.containingDependencyKey?.text ?: return
 
         val depToml = findDependencyTomlFile(containingArray, pkgName) ?: return
+        val originalArray = CompletionUtil.getOriginalElement(containingArray) ?: return
```

Comment:
Why do we require only original element here? To avoid `IntellijIdeaRulezzz ` in values?

Reply:
Yes, to remove it